test/change-stream.test.js

```diff
@@ -9,7 +9,8 @@ describe('PersistedModel.createChangeStream()', function() {
       var test = this;
       var app = loopback({ localRegistry: true });
       var ds = app.dataSource('ds', { connector: 'memory' });
-      this.Score = app.model('Score', {
+      var Score = app.registry.createModel('Score', { base: 'PersistedModel' });
```

Comment:
yeah, `PersistedModel` is set as base bydefault. I added it at the time for readibility, but no need, agreed.